rholang-parser/src/parser/parsing.rs

```diff
@@ -652,7 +652,17 @@ pub(super) fn node_to_ast<'ast>(
                     proc_stack.push(ast_builder.alloc_var_ref(var_ref_kind, var), span);
                 }
 
+                // Comments are named nodes when feature is enabled, but don't contribute to AST
+                #[cfg(feature = "named-comments")]
+                kind!("line_comment") | kind!("block_comment") => {
+                    // Skip comments - they're parsed for syntax highlighting but not part of AST
+                }
+
+                #[cfg(not(feature = "named-comments"))]
                 _ => unimplemented!(),
+
+                #[cfg(feature = "named-comments")]
+                _ => unreachable!("all node kinds should be handled when named-comments is enabled"),
```

Comment:
This is not true unfortunately. For one thing, there are some constructs which are unsupported (select). I'd just leave `unimplemented!()`